io.itch.App.yaml

```diff
@@ -0,0 +1,140 @@
+  - org.freedesktop.Platform.Compat.i386
+  - org.freedesktop.Platform.ffmpeg_full.i386
+  - org.winehq.Wine.gecko
+  - org.winehq.Wine.mono
+  - org.winehq.Wine.DLLs
+
+add-extensions:
+
+  org.freedesktop.Platform.Compat.i386.Debug:
+    directory: lib/debug/lib/i386-linux-gnu
+    no-autodownload: true
+    version: "22.08"
+
+  com.valvesoftware.Steam.Utility.gamescope:
+    add-ld-path: lib
+    directory: utils/gamescope
+
+modules:
+
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak
+        tag: v2022.04
+        commit: 55a60d110301e8dd37b631503c3524ab7baaf7aa
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+          
+  - name: itch-resources
+    buildsystem: simple
+    build-commands:
+      - install -D -t "${FLATPAK_DEST}/share/applications/" ./io.itch.App.desktop
+      - install -D -t "${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/" ./io.itch.App.svg
+      - install -D -t "${FLATPAK_DEST}/share/metainfo/" ./io.itch.App.metainfo.xml
+    sources:
+      - type: file
+        path: io.itch.App.svg
+        
+      - type: file
+        path: io.itch.App.desktop
+        
+      - type: file
+        path: io.itch.App.metainfo.xml
+
+  - name: itch
+    buildsystem: simple
+    build-commands:
+      - mv /run/build/itch/* /app/bin/
+    sources:
+      - type: script
+        dest-filename: itch-run
+        commands:
+          - zypak-wrapper.sh /app/bin/itch "$@"
+        
+      - type: archive
+        strip-components: 0
+        url: https://broth.itch.ovh/itch/linux-amd64/25.5.1/itch-linux-amd64.zip
+        sha256: fdd916c79beb90c19c7ee404e11398e5f2f5c7c402ae68910b4f70e6918071ed
+#        x-checker-data:
+#          type: rotating-url
+#          url: https://broth.itch.ovh/itch/linux-amd64/25.5.1/itch-linux-amd64.zip
+#          pattern: https://broth.itch.ovh/itch/linux-amd64/v([0-9.]+)/itch-linux-amd64.zip
+
+  - name: firejail
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://github.com/netblue30/firejail.git
+        tag: "0.9.70"
+        commit: b4b08d21cd95725c9d55dfdb6987fcc6d7893247
+#        x-checker-data:
+#          type: git
+#          tag-pattern: ^v([\\d.]+)$
+
+  - name: gamescope
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/utils/gamescope
+
+  - name: gamemode
+    buildsystem: meson
+    config-opts:
+#      - -Dwith-systemd=false
+#      - -Dwith-daemon=false
+      - -Dwith-examples=false
+      - -Dwith-util=false
+      - -Dwith-sd-bus-provider=no-daemon
+    sources: &gamemode_sources
+      - type: git
+        url: https://github.com/FeralInteractive/gamemode.git
+        tag: "1.7"
+        commit: 4dc99dff76218718763a6b07fc1900fa6d1dafd9
+#        x-checker-data:
+#          type: git
+#          tag-pattern: ^v([\\d.]+)$
+
+  - name: gamemoderun
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 data/gamemoderun -t /app/bin
```

Comment:
this doesnt need to be a separate module either.

Reply:
I do not know how to merge them, as they are different build systems

Reply:
this is one line in a post-install.

Reply:
Thanks for showing me how :heart: